<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCountryFilterBar</class>
 <widget class="QFrame" name="CCountryFilterBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Country filter bar</string>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <layout class="QGridLayout" name="gl_CountryFilterBar">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="2">
    <widget class="QLineEdit" name="le_IsoCode">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>ISO</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="swift::gui::filters::CFilterBarButtons" name="filter_Buttons">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="le_Name">
     <property name="placeholderText">
      <string>Name of country</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lbl_Name">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <spacer name="hs_CountryFilterBar">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lbl_CountryIso">
     <property name="toolTip">
      <string>Country ISO, e.g. DE, US, FR</string>
     </property>
     <property name="text">
      <string>Country ISO:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::filters::CFilterBarButtons</class>
   <extends>QFrame</extends>
   <header>gui/filters/filterbarbuttons.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_IsoCode</tabstop>
  <tabstop>le_Name</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
